docs/en_US/Compression/v2_pruning_algo.rst

```diff
@@ -273,6 +274,48 @@ User configuration for ADMM Pruner
 
 .. autoclass:: nni.algorithms.compression.v2.pytorch.pruning.ADMMPruner
 
+Movement Pruner
+---------------
+
+Movement pruner is an implementation of movement pruning.
+This is a pruning by step algorithm, the masks may change during each step.
```

Comment:
what is "step algorithm"?

Reply:
I want to say `pruning by step` algorithm... means this pruner will generate and apply masks during each optimizer step.

Reply:
I think every pruner is pruning step by step? What's the concrete meaning of `step` here?

Reply:
oh yes, it is easy to be misunderstood here, this means after each `optimizer.step()`, the model will be applied a new mask. I will update the docstring later.

Reply:
change to
`This is a "fine-pruning" algorithm, which means the masks may change during each fine-tuning step.`